app/src/main/java/com/codesoom/assignment/AppHttpHandler.java

```diff
@@ -0,0 +1,187 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.model.AppResponseEntity;
+import com.codesoom.assignment.model.HttpStatusCode;
+import com.codesoom.assignment.model.Task;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+
+import java.io.OutputStream;
+import java.io.IOException;
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.ByteArrayOutputStream;
+
+import java.net.URI;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.ArrayList;
+
+
+import java.util.NoSuchElementException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+public class AppHttpHandler implements HttpHandler {
+    private final static Logger logger = Logger.getGlobal();
+    private final String taskPath = "tasks";
+    private List<Task> tasks = new ArrayList<>();
+    private final ObjectMapper objectMapper = new ObjectMapper();
+
+
+    public AppHttpHandler() {
+        logger.setLevel(Level.INFO);
+    }
+
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+        String method = exchange.getRequestMethod();
+        URI uri = exchange.getRequestURI();
+        final int taskIdIdx = 2;
+        String uriPath = uri.getPath();
+        String[] uriPaths = uriPath.split("/");
+        String rootPath = uriPaths[1];
+        String taskId = "";
+        String contents = "";
+        if (uriPaths.length > taskIdIdx) {
+            taskId = uriPaths[taskIdIdx];
+        }
+
+        InputStream inputStream = exchange.getRequestBody();
+        String body = new BufferedReader(new InputStreamReader(inputStream))
+                .lines()
+                .collect(Collectors.joining("\n"));
+        logger.info("method : " + method + " /  uriPath : " + uriPath + " / body : " + body);
+
+        int httpStatusCode = HttpStatusCode.OK.getStatusCode();
```

Comment:
int 타입을 여기에서 사용하지 않고 이렇게 해보면 어떤 장점이 있을까요?
```suggestion
        HttpStatusCode httpStatusCode = HttpStatusCode.OK;
```

Reply:
가독성 부분에서 좀 더 나은 것 같습니다. 또한 해당 코드(getStatusCode())가 반복이 되는데 코드량도 줄 것 같네요